pom.xml

```diff
@@ -144,7 +144,7 @@
                         <configuration>
                             <referenceVersion>2.22ea3</referenceVersion>
                             <artifactsURI>https://teamcity.chronicle.software/repository/download</artifactsURI>
-                            <binaryCompatibilityPercentageRequired>95.1</binaryCompatibilityPercentageRequired>
+                            <binaryCompatibilityPercentageRequired>92.6</binaryCompatibilityPercentageRequired>
```

Comment:
I have manually checked the changes are ok. Mostly it is Enum converted to proper util classes meaning that values() etc. are missing. Also, internalized service implementations are safe to remove as they are not intended to be directly used.